### ISSUE TITLE
Breaking change in 1.2.0: some comments get duplicated

### ISSUE THREAD

**Issue by skeet70** [prettier/prettier#1348]
v1.1.0
```console
^(master) λ npm list prettier
my-app@2.24.1 /Users/skeet70/src/my-app
└─┬ prettier-eslint-cli@3.3.0
  └─┬ prettier-eslint@5.1.0
    └── prettier@1.1.0

^(master) λ npm run prettier                                                                                                                                 (my-app) 12:35:09

> my-app@2.24.1 prettier /Users/skeet70/src/my-app
> prettier-eslint --write "www/**/*.js"

78 files were unchanged
^(master) λ git diff                                                                                                                                         (my-app) 12:37:22
```

v1.2.0
```console
^(master) λ npm install prettier@1.2.0                                                                                                                       (my-app) 12:38:15
my-app@2.24.1 /Users/skeet70/src/my-app
└─┬ prettier-eslint-cli@3.3.0
  └─┬ prettier-eslint@5.1.0
    └─┬ prettier@1.2.0
      ├── ast-types@0.9.8
      ├── babylon@7.0.0-beta.8
      ├── flow-parser@0.43.0
      ├── get-stdin@5.0.1
      └── jest-validate@19.0.0

^(master) λ npm list prettier                                                                                                                                (my-app) 12:40:14
my-app@2.24.1 /Users/skeet70/src/my-app
└─┬ prettier-eslint-cli@3.3.0
  └─┬ prettier-eslint@5.1.0
    └── prettier@1.2.0

^(master) λ npm run prettier                                                                                                                                 (my-app) 12:40:21

> my-app@2.24.1 prettier /Users/skeet70/src/my-app
> prettier-eslint --write "www/**/*.js"

success formatting 3 files with prettier-eslint
75 files were unchanged
^(*master) λ git diff                                                                                                                                        (my-app) 12:40:40
```

```diff
diff --git a/www/js/config/ajax-status-handler.js b/www/js/config/ajax-status-handler.js
index 39dc4ca..982ff1f 100755
--- a/www/js/config/ajax-status-handler.js
+++ b/www/js/config/ajax-status-handler.js
@@ -44,7 +44,10
   /**
   * Set of default settings to be applied to model fetch calls in DAO layer.
   */
-  static get defaultSettings () {
+  static get defaultSettings/**
+  * Set of default settings to be applied to model fetch calls in DAO layer.
+  */
+  () {
diff --git a/www/js/library/tracker.js b/www/js/library/tracker.js
index 84b5580..b142d3b 100644
--- a/www/js/library/tracker.js
+++ b/www/js/library/tracker.js
@@ -80,7 +80,8
   }

   // I'd rather have this be private/just in the module and not exported, but then the call happens too early.
-  static get defaultTrackingParameters (): Object {
+  static get defaultTrackingParameters// I'd rather have this be private/just in the module and not exported, but then the call happens too early.
+  (): Object {

diff --git a/www/js/views/explore.js b/www/js/views/explore.js
index b918a2c..64989aa 100644
--- a/www/js/views/explore.js
+++ b/www/js/views/explore.js
@@ -122,7 +122,9
     if (
-      this.options.criteria != null && this.options.criteria.dimensions != null && this.options.criteria.metrics != null
+      this.options.criteria != null &&
+      this.options.criteria.dimensions != null &&
+      this.options.criteria.metrics != null
     ) {
```

I would expect that those comments stay where they were originally. The giant boolean change is a good one 😄 .

**Comment by vjeux** [comment]
Omg, you are so quick! Thanks a lot for the report, I'm going to fix this asap!

**Comment by vjeux** [comment]
I just published 1.2.1 to fix this, so sorry about it and thanks for reporting it blazingly fast!

**Comment by skeet70** [comment]
@vjeux no problem, thanks for fixing it so quickly!
